/mnt/c/Users/mngyn/hemodialysis_trends_pipeline/airflow_env/venv/lib/python3.10/site-packages/flask_limiter/extension.py:336 UserWarning: Using the in-memory storage for tracking rate limits as no storage was explicitly specified. This is not recommended for production use. See: https://flask-limiter.readthedocs.io#configuring-a-storage-backend for documentation about configuring the storage backend.
[[34m2025-04-21T11:45:43.365+0800[0m] {[34moptions.py:[0m83} WARNING[0m - The swagger_ui directory could not be found.
    Please install connexion with extra install: pip install connexion[swagger-ui]
    or provide the path to your local installation by passing swagger_path=<your path>
[0m
[[34m2025-04-21T11:45:43.384+0800[0m] {[34moptions.py:[0m83} WARNING[0m - The swagger_ui directory could not be found.
    Please install connexion with extra install: pip install connexion[swagger-ui]
    or provide the path to your local installation by passing swagger_path=<your path>
[0m
[2025-04-21 11:45:43 +0800] [460] [INFO] Starting gunicorn 21.2.0

Error: Already running on PID 370 (or pid file '/mnt/c/Users/mngyn/hemodialysis_trends_pipeline/airflow/airflow-webserver.pid' is stale)

  ____________       _____________
 ____    |__( )_________  __/__  /________      __
____  /| |_  /__  ___/_  /_ __  /_  __ \_ | /| / /
___  ___ |  / _  /   _  __/ _  / / /_/ /_ |/ |/ /
 _/_/  |_/_/  /_/    /_/    /_/  \____/____/|__/
Running the Gunicorn Server with:
Workers: 4 sync
Host: 0.0.0.0:8080
Timeout: 120
Logfiles: - -
Access Logformat: 
=================================================================
[[34m2025-04-21T11:47:03.244+0800[0m] {[34mwebserver_command.py:[0m222} ERROR[0m - No response from gunicorn master within 120 seconds[0m
[[34m2025-04-21T11:47:03.259+0800[0m] {[34mwebserver_command.py:[0m223} ERROR[0m - Shutting down webserver[0m
